types/ckeditor__ckeditor5-core/ckeditor__ckeditor5-core-tests.ts

```diff
@@ -128,34 +142,43 @@ delete command.isEnabled;
 const context = new Context();
 const contextWithConfig = new Context({ foo: 'foo' });
 context.destroy().then(() => {});
-contextWithConfig.initPlugins().then(plugins => plugins.map(plugin => plugin.pluginName));
+contextWithConfig.initPlugins().then(loaded => {
+    loaded.forEach(Plugin => {
+        class Foo extends Plugin {}
+        new Foo(editor);
+    });
+});
 
 /**
  * ContextPlugin
  */
-const CPlugin = new ContextPlugin(context) && new ContextPlugin(editor);
-const afterInitPromise = CPlugin.afterInit?.();
-if (afterInitPromise != null) {
-    afterInitPromise.then(() => {});
-}
-
 class MyCPlugin extends ContextPlugin {
-    get pluginName() {
-        return 'MyCPlugin';
-    }
-
+    readonly pluginName: 'MyCPlugin';
```

Comment:
We need `target: es6` in `tsconfig` for getters to work, and changing `tsconfig` requires a DT mantainer to review it.